src/main/java/com/zendesk/maxwell/schema/SchemaCapturer.java

```diff
@@ -110,8 +114,26 @@ private void captureTable(Table t) throws SQLException {
 			if ( r.getString("COLUMN_KEY").equals("PRI") )
 				t.pkIndex = i;
 
-			t.getColumnList().add(ColumnDef.build(t.getName(), colName, colEnc, colType, colPos, colSigned));
+			if ( colType.equals("enum") || colType.equals("set")) {
+				String expandedType = r.getString("COLUMN_TYPE");
+
+				enumValues = extractEnumValues(expandedType);
+			}
+
+			t.getColumnList().add(ColumnDef.build(t.getName(), colName, colEnc, colType, colPos, colSigned, enumValues));
 			i++;
 		}
 	}
+
+	private static String[] extractEnumValues(String expandedType) {
+		String[] enumValues;
+		Matcher matcher = Pattern.compile("(enum|set)\\((.*)\\)").matcher(expandedType);
```

Comment:
it allows you to pass the backslash into the regexp engine to escape the parens: the string reads literally `(enum|set)\((.*)\)`